salt/pillar/top.sls

```diff
@@ -2,4 +2,4 @@ base:
     '*':
         - elife
         - peerscout
-        - peerscout-secrets
+        #- peerscout-secrets
```

Comment:
This is needed, but must be in `.gitignore` since contains credentials; see `README`